example/src/test/scala/other/ImportExample.scala

```diff
@@ -39,16 +35,16 @@ class ImportExample extends AnyFunSuite with Configuration with FunSuiteDiscipli
     // do not compile because scala cannot infer which instance of Each is required
     illTyped("""each.modify(List(1,2,3), _ + 1)""")
 
-    each[List[Int], Int].modify(_ + 1)(List(1, 2, 3)) shouldEqual List(2, 3, 4)
+    assertEquals(each[List[Int], Int].modify(_ + 1)(List(1, 2, 3)), List(2, 3, 4))
   }
 
   test("monocle.std.all._ brings all polymorphic Optic instances in scope for standard Scala classes") {
     import monocle.function.all._
 
     // do not compile because Head instance for HList is not in scope
-    illTyped("""head[Int :: HNil, Int].modify(1 :: HNil, _ + 1) shouldEqual (2 :: HNil)""")
+    illTyped("""assertEquals(head[Int :: HNil, Int].modify(1 :: HNil, _ + 1),  (2 :: HNil))""")
```

Comment:
On a second pass, we may want to use `compileErrors` from munit instead of illTyped